src/Storage/Migration/v0.12/01-setup-tables.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE storage.a2xsls ADD COLUMN IF NOT EXISTS isportrait BOOL NOT NULL DEFAULT true;
+DROP FUNCTION IF EXISTS storage.reada2xsls(_org TEXT, _app TEXT, _lformid INT, _language TEXT, _xsltype INT);
```

Comment:
This is not backwards compatible. Wondering if `CREATE OR REPLACE` is good enough or if we need a new version.

Reply:
Backwards compability is not an issue because this PR should be released long before it affects other than team dataflytt members.

Reply:
Still not sure why we have DROP FUNCTION here when we have CREATE OR REPLACE later.

Reply:
It's just to remove the old version with fewer parameters - REPLACE doesn't touch the old one. Normally (when it affects real users) we must postpone the delete until the next release, and it's normally forgotten.